.github/workflows/build-and-test.yaml

```
@@ -36,7 +36,18 @@ jobs:
           target-packages: ${{ steps.get-self-packages.outputs.self-packages }}
           build-depends-repos: build_depends.repos
 
+      - name: Check file existence
+        id: check_files
+        run: |
+          set -e
+          if [ -f "lcov/total_coverage.info" ] || [ -f "coveragepy/.coverage" ]; then
+            echo ::set-output name=files_exists::"true"
```

`files-exists` is strange. `file-exists`, `files-exist`, or just `exists` is good, I think.

```suggestion
            echo ::set-output name=exists::"true"
```

---

fc859c7